R/uniDimBayesianOutputTables.R

```diff
@@ -249,3 +250,80 @@
 
   return()
 }
+
+
+.BayesianFitMeasuresTable <- function(jaspResults, model, options) {
+
+  if (!is.null(.getStateContainerB(jaspResults)[["fitTable"]]$object))
+    return()
+
+  fitTable <- createJaspTable(gettextf("Fit Measures for the Single-Factor Model"))
+
+  fitTable$dependOn(options = c("omegaScale", "fitMeasures", "fitCutoffSat", "fitCutoffNull", "pointEst"))
+
+  cred <- format(100 * options[["credibleIntervalValueItem"]], digits = 3, drop0trailing = TRUE)
+
+  fitTable$addColumnInfo(name = "measure", title = gettext("Fit measure"), type = "string")
+  fitTable$addColumnInfo(name = "pointEst", title = gettext("Point estimate"), type = "number")
+  fitTable$addColumnInfo(name = "cutoff",
+                         title = gettextf("Relative to cutoff"), type = "number")
+
+
+  if (options[["omegaScale"]] && options[["fitMeasures"]] && is.null(model[["empty"]])) {
+
+    pointEsts <- sapply(model[["fitMeasures"]], get(options[["pointEst"]]))
```

Comment:
In general, `get` is a bit iffy. Can you write a helper function that does something like:

```r
getPointEstFun <- function(pointEstFunString) {
  return switch(pointEstFunString,
    "mean"   = mean,
    "median" = median,
    # etc.
    # default case
    stop("getPointEstFun has no case for value: ", pointEstFunString)
  )
}
```